src/topology/vector_bundle/hom.lean

```diff
@@ -85,7 +85,6 @@ variables (F₂ : Type*) [normed_add_comm_group F₂][normed_space 𝕜₂ F₂]
 namespace topological_vector_bundle
 
 variables {F₁ E₁ F₂ E₂} (e₁ e₁' : trivialization 𝕜₁ F₁ E₁) (e₂ e₂' : trivialization 𝕜₂ F₂ E₂)
-variables [ring_hom_isometric σ]
```

Comment:
What is the reason for moving this?

Reply:
Well there are theorems in the section which no longer require `σ` to be isometric, so if I keep this here the linter complains. So I removed it here and added it manually to the theorems requiring it.